modules/client-configuration/pages/clients-rh.adoc

```diff
@@ -155,161 +135,131 @@ subscription-manager repos
-Navigate to the [guimenu]``Repositories`` tab, then click [guimenu]``Sync`` and check [systemitem]``Sync Status``.
-endif::[]
-* Check the synchronization log file at the command prompt:
-+
-----
-tail -f /var/log/rhn/reposync/<channel-label>.log
-----
-+
-Each child channel will generate its own log during the synchronization progress.
-You will need to check all the base and child channel log files to be sure that the synchronization is complete.
 
 
-[NOTE]
-====
-{rhel} channels can be very large.
-Synchronization can sometimes take several hours.
-====
+== Add Software Channels
 
+Before you register {redhat} clients to your {productname} Server, check that you have the {redhat} product enabled, and the required channels are fully synchronized.
 
-When you have created the custom channels and synchronized them with the repositories, you can create child channels.
+ifeval::[{suma-content} == true]
 
-.Procedure: Creating Child Channels
+Your {susemgr} subscription entitles you to the tools channels for {sleses} (also known as {redhat} Expanded Support or RES).
+You must use the client tools channel to create the bootstrap repository.
+This procedure applies to both Salt and traditional clients.
 
-. On the {productname} Server {webui}, navigate to menu:Software[Manage > Channels].
-. Click btn:[Create Channel] and set these parameters for the entitlement certificate.
-Ensure you use the correct {rhela} version:
-* In the [guimenu]``Channel Name`` field, type [systemitem]``RHEL 7 x86_64``.
-* In the [guimenu]``Channel Label`` field, type [systemitem]``rhel7-x86_64-extras``.
-* In the [guimenu]``Parent Channel`` field, select [systemitem]``rhel7-x86_64-server``.
-* In the [guimenu]``Architecture`` field, select [systemitem]``x86_64``.
-* In the [guimenu]``Repository Checksum Type`` field, select [systemitem]``sha1``.
-* In the [guimenu]``Channel Summary`` field, type [systemitem]``RHEL 7 x86_64 Extras``.
-* In the [guimenu]``Organization Sharing`` field, select [systemitem]``Public``.
-. Click btn:[Create Channel].
-. Navigate to the [guimenu]``Repositories`` tab, check the appropriate repository, and click btn:[Update repositories].
-. OPTIONAL: Navigate to the [guimenu]``Sync`` tab to set a recurring schedule for synchronization of this repository.
-. Click btn:[Sync Now] to begin synchronization immediately.
 
+ifeval::[{suma-content} == true]
 
-[NOTE]
-====
-{rhel} channels can be very large.
-Synchronization can sometimes take several hours.
-====
+The channels you need for this procedure are:
 
+[[redhat-channels-wizard]]
+[cols="1,1,1", options="header"]
+.Red Hat Channels - WebUI
+|===
+| OS Version | Base Channel                     | Pool Channel
+| {redhat} 6 | RHEL6 Base x86_64                | RHEL6-Pool for x86_64
+| {redhat} 7 | RHEL7 Base x86_64                | RHEL7-Pool for x86_64
+| {redhat} 8 | RHEL or SLES ES or CentOS 8 Base | RHEL8-Pool for x86_64
+|===
```

Comment:
here we have what is needed earlier. Which brings me back to the question: what do you want to describe?

Reply:
Basically, each of these tables is just trying to give people the information they need to complete the procedure. So if I'm in the WebUI, I'm trying to add RH8 channels, I want to be able to look up the table, and know what to search for and what boxes to check. At that stage, possibly the subtle differences between channels and products are a bit academic?

Reply:
I think not. You are on a page which let you see and add products. When you add that, and you go later to software => Channels and look for it, you will see different names. 

When you mirror a product, you get a bunch of channels. I think it is good to tell them what channels you get and how to use them. Especially for RedHat (and ubuntu 16/18) where we only provide incomplete setup.